NitroxModel/Discovery/InstallationFinders/EnvironmentGameFinder.cs

```diff
@@ -1,30 +1,29 @@
-﻿using System;
+﻿namespace NitroxModel.Discovery.InstallationFinders;
```

Comment:
Namespace declaration should be below usings (using statements should always come first)